README.md

```diff
@@ -143,9 +143,14 @@ and
 Order Multiple Teacher Sets Configuration
 ========================
 ```
-MAXIMUM_COPIES_REQUESTABLE :5  - This is a configuration value in AWS ElasticBeanstalk. In future if anyone want to change the value of maximum teacherset orders, we can update in AWS ElasticBeanstalk Configuration.
+MAXIMUM_COPIES_REQUESTABLE: 5  - This is a configuration value in AWS ElasticBeanstalk. In future if anyone want to change the value of maximum teacherset orders, we can update in AWS ElasticBeanstalk Configuration.
 ```
 
+Show Maintenance Banner Configuration
+========================
+```
+SHOW_MAINTENANCE_BANNER: TRUE  - This is a configuration value in AWS ElasticBeanstalk. In future if anyone want to change the value of maintenanct banner, we can update in AWS ElasticBeanstalk Configuration.
```

Comment:
This is so that someone can turn on the banner, right? To change the value we have to modify the HTML. So let's say:
"This parameter should be set to the string`true` to turn on the banner, which is set in  app/views/layouts/angular.html.erb and  app/views/layouts/application.html.erb."

Alternatively, we could create a second variable called `MAINTENANCE_BANNER_TEXT` which would actually contain the value to display. That would be useful. Then the sentence would be 
"This parameter should be set to the string`true` to turn on the banner, which is set in  the `MAINTENANCE_BANNER_TEXT` environment variable."